blog/17-2-23-Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity.md

```diff
@@ -0,0 +1,1004 @@
+---
+title: Build a Full Stack Coffee Dapp With Celo Composer and Solidity
+description: This tutorial will take you through a step-by-step guide on how to create a frontend and backend (Smart Contract) dApp explaining how to create a decentralized version of Buy Me A Coffee.
+authors:
+  - name: Glory Agatevure
+    title: Blockchain Engineer | Technical Writer
+    url: https://www.linkedin.com/in/glory-agatevure-47a222ab/
+    image_url: https://avatars.githubusercontent.com/u/23031920?s=400&u=99aba40f495b86f3936a53e06dc7fbbc278b98ad&v=4
+tags: ['celosage','composer','celo','celowallet','reactcelo','contractkit','dapp','valora','typescript', 'smartcontract', 'solidity', 'nextjs', 'intermediate', 'advanced']
+hide_table_of_contents: true
+slug: /tutorials/Build-a-Full-Stack-Coffee-Dapp-With-Celo-Composer-and-Solidity
+
```

Comment:
Add header image

Reply:
I have added the path to the header image on the users.tsx file.

Is there anything that is not correct on that file.